ui/components/Connect/connections/helm-connect/StepperContent.js

```diff
@@ -0,0 +1,369 @@
+import React from 'react';
+import { Alert, AlertTitle, Button, Stack } from '@material-ui/core';
+import { ConnectAppContent, FinishContent, SelectRepositoryContent } from './constants';
+import Router, { useRouter } from 'next/router';
+import { useTheme } from '@material-ui/core';
+import { useNotification } from '../../../../utils/hooks/useNotification';
+import { EVENT_TYPES } from '../../../../lib/event-types';
+import { postHelmInstall } from './utils';
+import { VerifyConnectionKind } from '../../helpers/common.js';
+import StepperContent from '../../stepper/StepperContentWrapper';
+import RJSFWrapper from '../../../MesheryMeshInterface/PatternService/RJSF_wrapper';
+// import { useGetSchemaQuery, useUpsertApplicationMutation } from "@/api/api";
+import DesignsIcon from '../../../../assets/icons/DesignIcon';
+import { promisifiedDataFetch } from '../../../../lib/data-fetch';
+
+export const ConnectApp = ({ handleNext }) => {
+  const [isConnected, setIsConnected] = React.useState(false);
+  const formRef = React.createRef();
+  const formStateRef = React.createRef();
+  const [schema, setSchema] = React.useState({});
+  const [isSchemaFetched, setIsSchemaFetched] = React.useState(false);
+  const [formData, setFormData] = React.useState({});
+
+  const handleCallback = () => {
+    if (formRef.current && formRef.current.validateForm()) {
+      // Check if the URL is valid via an API 
+      axios
```

Comment:
Axios?

We don’t use axios in Meshery UI but fetchAPI.

Reply:
Have we started using RTK in MesheryUI?
RTK by default uses fetch API, so if we have not started you can initiate usage of RTK , you can do that (in this PR or separately)

Reply:
Yes we are using RTK now I have created all RTK APIs